src/interaction/interactiveTarget.js

```diff
@@ -42,5 +49,11 @@ module.exports = {
      * @todo Needs docs.
      * @private
      */
-    _touchDown: false
+    _touchDown: false,
+    
+    /**
+     * Used to track the global start position of a touchdown event on this target.
+     * @private
+     */
+    _touchDownPoint: new core.Point()
```

Comment:
Yes this is because of multitouch, although there's no check right now if it was the same touch event... An idea would be to make a pool of data objects which can store the touch related info and are related to an touch event through the touch event identifier. We can then fill them with info on touchstart, update them on touchmove and release them on touchend after we used the info.